tests/unit/ui/components/TestRating.js

```diff
@@ -27,6 +40,19 @@ function makeFakeEvent() {
   };
 }
 
+function signInAndDispatchSavedReview({ siteUserId, reviewUserId }) {
```

Comment:
I borrowed this code from [`TestAddonReviewListItem`](https://github.com/mozilla/addons-frontend/blob/master/tests/unit/amo/components/TestAddonReviewListItem.js#L101) , but I shortened it (and inlined some other functions it calls) for use in this file. I considered moving the functions that support this into a helper for use by both of these tests, but this seemed like a reasonable approach as well.